Shared/ViewModels/VideoPlayerViewModel/VideoPlayerViewModel.swift

```diff
@@ -151,7 +151,8 @@ final class VideoPlayerViewModel: ViewModel {
 	}
 
 	func setSeconds(_ seconds: Int64) {
-		let videoDuration = item.runTimeTicks!
+        guard let runTimeTicks = item.runTimeTicks else { return }
```

Comment:
prevents a crash that could occur here with the implicitly unwrapped optional. Ran into this with LiveTV.